common/lib/capa/capa/templates/vsepr_input.html

```diff
@@ -21,13 +21,13 @@
    />
 
       <p class="status" aria-describedby="input_${id}">
-      ${status.display_name}
+         <span class="sr">${status.display_name}</span>
       </p>
 
   <p id="answer_${id}" class="answer"></p>
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="molecule_select_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
It looks like this input type (vsepr_input) is somewhat tied to crystallography, and dates from the same time period (2012). There appear to be no references to it.

@sstack22 OK to delete as part of deleting crystallography?

@alisan617 let's not worry about adding aria-describedby to this template.